SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+Security alerts related to the hosting and delivery of the ecips.ethereumclassic.org web site will be addressed and will be taken by either the ecip editors or org admins. Security concerns with I individual ecips should be addressed in the `discussion to` locations.
+Security alerts will not be made publicly until mitigating solutions are identified. Once a remedy is found, dependably will automate the repair process.
+
+This site is free and open source in accordance with the site license which can be found at LICENSE.md or COPY.md
```

Comment:
I don't think the licensing issue belongs here. I would suggest removing this avoiding any possibility to get this controversial.

Reply:
I didn't think about that. This is to cover the website.